From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mirko Bonadei <mbonadei@chromium.org>
Date: Tue, 10 Mar 2020 06:04:07 +0000
Subject: - Point usrsctp to a68325e7d9ed844cc84ec134192d788586ea6cc1.

Bug: 1059349
Change-Id: I451873d552b6fe0ca6cd481385042bf37a05fbfd
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2093604
Reviewed-by: Guido Urdaneta <guidou@chromium.org>
Cr-Commit-Position: refs/branch-heads/4044@{#735}
Cr-Branched-From: a6d9daf149a473ceea37f629c41d4527bf2055bd-refs/heads/master@{#737173}

diff --git a/DEPS b/DEPS
index 4efda890e2a2b9e4597a03a435a5fc0b716f6564..6a46b05c750c82fc43fe118b2493c78518f271a9 100644
--- a/DEPS
+++ b/DEPS
@@ -1388,7 +1388,7 @@ deps = {
   },
 
   'src/third_party/usrsctp/usrsctplib':
-    Var('chromium_git') + '/external/github.com/sctplab/usrsctp' + '@' + '7a8bc9a90ca96634aa56ee712856d97f27d903f8',
+    Var('chromium_git') + '/external/github.com/sctplab/usrsctp' + '@' + 'a68325e7d9ed844cc84ec134192d788586ea6cc1',
 
   # Display server protocol for Linux.
   'src/third_party/wayland/src': {
